src/server.js

```diff
@@ -17,12 +19,27 @@ app.use(cors());
 
 app.use(morgan("combined"));
 
+// Used by certain endpoints for verification
+async function verify(token, email) {
+  const ticket = await client.verifyIdToken({
+    idToken: token,
+    audience: process.env.REACT_APP_GOOGLE_CLIENT_ID
+  });
+  const payload = ticket.getPayload();
+  return payload.email == email;
+}
+
 // Test API method 
 app.get("/test", (req, res) => {
   res.status(200).send("Hello World!");
 });
 
 // Add API methods here
+app.post("/users", (req, res) => {
+  
```

Comment:
you can file yourself an issue and link this here with a //TODO (issue 1) or something.

Then when you actually fill this in, you'll link that pull request with the issue. This is so dangling items like this doesn't get forgotten.

Reply:
done